migrations/Migration20240709101515.ts

```diff
@@ -0,0 +1,41 @@
+import { Migration } from '@mikro-orm/migrations';
+
+export class Migration20240709101515 extends Migration {
+
+  async up(): Promise<void> {
+    this.addSql('alter table "personenkontext" drop constraint "personenkontext_person_id_id_foreign";');
+
+    this.addSql('alter type "organisations_typ_enum" add value if not exists \'SONSTIGE ORGANISATION / EINRICHTUNG\';');
+
+    this.addSql('alter table "personenkontext" drop constraint "personenkontext_person_id_id_organisation_id_rolle_id_unique";');
+
+    this.addSql('alter table "personenkontext" alter column "rolle_id" drop default;');
+    this.addSql('alter table "personenkontext" alter column "rolle_id" type uuid using ("rolle_id"::text::uuid);');
+    this.addSql('alter table "personenkontext" alter column "rolle_id" set not null;');
+    this.addSql('alter table "personenkontext" rename column "person_id_id" to "person_id";');
+    this.addSql('alter table "personenkontext" add constraint "personenkontext_person_id_foreign" foreign key ("person_id") references "person" ("id") on delete cascade;');
+    this.addSql('alter table "personenkontext" add constraint "personenkontext_rolle_id_foreign" foreign key ("rolle_id") references "rolle" ("id") on update cascade;');
+    this.addSql('alter table "personenkontext" add constraint "personenkontext_person_id_organisation_id_rolle_id_unique" unique ("person_id", "organisation_id", "rolle_id");');
+
+    this.addSql('alter table "service_provider" add column "keycloak_group" varchar(255) null, add column "keycloak_role" varchar(255) null;');
+  }
+
+  override async down(): Promise<void> {
+    this.addSql('alter table "personenkontext" drop constraint "personenkontext_person_id_foreign";');
+    this.addSql('alter table "personenkontext" drop constraint "personenkontext_rolle_id_foreign";');
+
+    this.addSql('alter type "organisations_typ_enum" add value if not exists \'"SONSTIGE ORGANISATION / EINRICHTUNG"\';');
+
+    this.addSql('alter table "personenkontext" drop constraint "personenkontext_person_id_organisation_id_rolle_id_unique";');
+
+    this.addSql('alter table "personenkontext" alter column "rolle_id" drop default;');
+    this.addSql('alter table "personenkontext" alter column "rolle_id" type uuid using ("rolle_id"::text::uuid);');
+    this.addSql('alter table "personenkontext" alter column "rolle_id" drop not null;');
+    this.addSql('alter table "personenkontext" rename column "person_id" to "person_id_id";');
```

Comment:
Eine Frage aus Küriosität und zwar das Feld person_id wurde  in der Entity schon unbenannt und wir haben schon ein Paar Migrationen hinter uns, gibt es einen Grund warum es bei der Migration noch in der Tabelle  personenkontext unbenannt wird? :)

Reply:
Also eigentlich sollte sie lieber person_id heißen. Da schon einige Migrationen im ausgeführt wurden und auch gemerged werden musste, ist das aktuell das Ergebnis. Das ist vielleicht so nicht sinnvoll, das muss ich mir mal im Detail anschauen, was da weshalb zusammengekommen ist.